docs/articles/baseplatform/booting-qemu-image.rst

```diff
@@ -41,6 +44,56 @@ following command:
       -m 512 \
       --append "vga=0 uvesafb.mode_option=1280×720-32 root=/dev/hda console=ttyS0 rw mem=512M oprofile.timer=1 " -serial stdio
 
+Static IP address configuration:
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Static IP address facilitates more network services for development and
```

Comment:
"A static IP address facilitates development of automation and testing scripts."